File: python/hanb.py
#!//usr/local/bin/python3

edgeboard = """
        a   a   a   a   a
      a   a   a   a   a   a
    a   a   a   a   a   a   a
  a   a   a   a   a   a   a   a
a   a   a   a   a   a   a   a   a
  a   a   a   a   a   a   a   a
    a   a   a   a   a   a   a
      a   a   a   a   a   a
        a   a   a   a   a
"""

pointboard = """
                    a
               a         a
          a         a         a
     a         a         a         a
a         a         a         a         a
     a         a         a         a
a         a         a         a         a
     a         a         a         a
a         a         a         a         a
     a         a         a         a
a         a         a         a         a
     a         a         a         a
a         a         a         a         a
     a         a         a         a
          a         a         a
               a         a
                    a
"""

def print_board(board):
    print(board)

def main():
    print_board(edgeboard)
    print_board(pointboard)

if __name__ == "__main__":
    main()

File: c/hanb.c
#include <stdio.h>

int print_board(int board_rotation)
{
char *edgeboard = "\
        a   a   a   a   a         \n\
      a   a   a   a   a   a       \n\
    a   a   a   a   a   a   a     \n\
  a   a   a   a   a   a   a   a   \n\
a   a   a   a   a   a   a   a   a \n\
  a   a   a   a   a   a   a   a   \n\
    a   a   a   a   a   a   a     \n\
      a   a   a   a   a   a       \n\
        a   a   a   a   a         \n\
";

char *pointboard = "\
                    a                     \n\
               a         a                \n\
          a         a         a           \n\
     a         a         a         a      \n\
a         a         a         a         a \n\
     a         a         a         a      \n\
a         a         a         a         a \n\
     a         a         a         a      \n\
a         a         a         a         a \n\
     a         a         a         a      \n\
a         a         a         a         a \n\
     a         a         a         a      \n\
a         a         a         a         a \n\
     a         a         a         a      \n\
          a         a         a           \n\
               a         a                \n\
                    a                     \n\
";

if(board_rotation == 0)
    {
    printf("%s", edgeboard);
    }
if(board_rotation == 1)
    {
    printf("%s", pointboard);
    }

return 0;
}

int main()
{
print_board(0);
print_board(1);

return 0;
}
